org.kde.minuet.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "org.kde.minuet",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "minuet",
+    "rename-icon": "minuet",
+    "finish-args": [
+        "--socket=pulseaudio", 
+        "--share=ipc", 
+        "--socket=x11", 
+        "--socket=wayland",
+        "--device=dri"
+    ],
+
+    "modules": [
+        {
+            "name": "fluidsynth",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/FluidSynth/fluidsynth/archive/v2.1.4.tar.gz",
+                    "sha256": "6e4f3e4a957d03812765bd0e400f61cd6c9fdb4bf10970e5cdaea8487d54b250"
+                }
+            ]
+        },
+        {
+            "name": "minuet",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                { 
+                    "type": "archive", 
+                    "url": "https://download.kde.org/stable/release-service/20.04.2/src/minuet-20.04.2.tar.xz",
```

Comment:
should be 20.04.3

Reply:
https://github.com/flathub/flathub/pull/1664/commits/29058dddcfd2962ee366f30499d830f52640561d